docker-compose.yml

```diff
@@ -58,13 +58,13 @@ services:
   storage:
     image: fsouza/fake-gcs-server
     ports:
-      - "4443:4443"
-    command: -scheme http
+      - "8080:8080"
+    command: ["-scheme", "http", "-port", "8080"]
     volumes:
       - storage_data:/storage
 
   postgres:
-    image: postgres:14
+    image: postgres:14.1-alpine
```

Comment:
You are pinning 14.1 (hence downgrading the postgres version). I'd rather leave 14. (`14-alpine` is fine too)